bluemira/base/designer.py

```diff
@@ -57,7 +57,11 @@ def run(self) -> _DesignerReturnT:
         pass
 
     def _init_params(self, params: Union[Dict, _ParameterFrameT]) -> _ParameterFrameT:
-        if isinstance(params, dict):
+        if params is None:
+            if self.param_cls is None:
+                return None
+            return self.param_cls.from_dict({})
+        elif isinstance(params, dict):
```

Comment:
Do you think we should raise in the case of `params is None and param_cls is not None`? I think all it will mean is an error being raised somewhere else down the line.

In any case, this logic should be consistent with the logic in `Builder._init_params`, so one of them is going to need updating. We should probably refactor the logic out - needn't be in this PR though

Reply:
I was thinking about whether someone might want fallback defaults in their specific parameter class (therefore no error will be raised). I suppose that should really not be allowed and they should just update their config file